packages/quick_actions/quick_actions_ios/example/ios/RunnerTests/Mocks/MockShortcutItemProvider.swift

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import UIKit
 @testable import quick_actions_ios
```

Comment:
[swift format error](https://ci.chromium.org/ui/p/flutter/builders/try/Mac_x64%20check_podspecs/12657/overview):
```suggestion
import UIKit

@testable import quick_actions_ios
```